boxsdk/object/user.py

```diff
@@ -1,16 +1,128 @@
 # coding: utf-8
-
 from __future__ import unicode_literals
 
+import json
+
 from .base_object import BaseObject
 from ..pagination.limit_offset_based_object_collection import LimitOffsetBasedObjectCollection
+from ..pagination.marker_based_object_collection import MarkerBasedObjectCollection
+from ..util.api_call_decorator import api_call
 
 
 class User(BaseObject):
     """Represents a Box user."""
 
     _item_type = 'user'
 
+    @api_call
+    def add_email_alias(self, email):
+        """
+        Adds a new email alias to the given user's account.
+
+        :param email:
+            The email alias to add to the user.
+        :type email:
+            `unicode`
+        :returns:
+            The new email alias object
+        :rtype:
+            :class:`EmailAlias`
+        """
+        url = self.get_url('email_aliases')
+        body = {
+            'email': email,
+        }
+        response = self._session.post(url, data=json.dumps(body)).json()
+        return self.translator.translate(
+            session=self._session,
+            response_object=response,
+        )
+
+    @api_call
+    def get_email_aliases(self, limit=None, fields=None):
+        """
+        Gets an list of email aliases for a user.
+
+        :param limit:
+            The maximum number of users to return. If not specified, the Box API will determine an appropriate limit.
+        :type limit:
+            `int` or None
+        :param fields:
+            List of fields to request
+        :type fields:
+            `Iterable` of `unicode`
+        :returns:
+            An iterator of the user's email aliases
+        :rtype:
+            :class:`BoxObjectCollection`
+        """
+        return MarkerBasedObjectCollection(
+            session=self._session,
+            url=self.get_url('email_aliases'),
+            limit=limit,
+            marker=None,
+            fields=fields,
+            return_full_pages=False,
+        )
+
+    @api_call
+    def remove_email_alias(self, email_alias):
+        """
+        Remove an email alias from the user.
+
+        :param email_alias:
+            The email alias to remove.
+        :type email_alias:
+            :class:`EmailAlias`
+        :returns:
+            Whether the removal succeeded.
+        :rtype:
+            `bool`
+        """
+        url = self.get_url('email_aliases', email_alias.object_id)
+        response = self._session.delete(url, expect_json_response=False)
+        return response.ok
```

Comment:
This should just be `return email_alias.delete()`

Is there any reason this method needs to exist?

Reply:
Oh, I didn't notice that. I guess this is fine for now then.

Or we can move this implementation into an override of `delete()` on EmailAlias, and have it enforce that you give a `user` input.